### PR TITLE
Fix column order in data exported from matplotlib linegraph with `--data-format tsv --export`

### PR DESCRIPTION
**Pull request by vladsavelyev**
Fixes https://github.com/ewels/MultiQC/issues/2140

https://github.com/ewels/MultiQC/pull/1891 insanely optimised exporting of plot data, however we accidentally lost the sorting order of columns in the output TSV. E.g.

```
multiqc -f -v -m dragen_fastqc test_data/data/modules/dragen_fastqc --data-format tsv --export
```
 
In `multiqc_data/mqc_dragenqc_per_sequence_quality_scores_plot_1.txt`:

Screenshot 2023-10-19 at 22 05 08

The `sort_cols=True` option to `write_data_file` still works, however, keeping the default order is broken.

This PR fixes the issue:

Screenshot 2023-10-19 at 22 03 50
<br/><br/>

### CODE REVIEW

**Comment by ewels** [comment]
After discussion: better to maintain the order of columns from the module, rather than sorting in the function that saves the file.

**Comment by vladsavelyev** [comment]
Updated the PR to keep the `sort_cols` flag - please review again.

**Review by ewels — APPROVED** [review]
Great 👍🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### MultiQC updates
 
+- Fix export TSV from a matplotlib linegraph plot ([#2143](https://github.com/ewels/MultiQC/pull/2143))
```
Comment (ewels) [inline CHANGELOG.md:7]:
```suggestion
- Fix column sorting in exported TSV files from a matplotlib linegraph plot ([#2143](https://github.com/ewels/MultiQC/pull/2143))
```

**Pull request merged**